Core/clim-core/standard-presentations.lisp

```diff
@@ -129,28 +129,31 @@
 (define-presentation-type blank-area ()
   :inherit-from t)
 
+;;; Do other slots of this have to be bound in order for this to be
+;;; useful?  Guess we'll see.
+;;;
+;;; This variable is specified to be a constant.
```

Comment:
earmuffs suggests this is a mistake in spec. Lispworks CLIM guide calls it a variable.

----
![Screenshot from 2020-07-02 19-48-35](https://user-images.githubusercontent.com/10996250/86391975-47090800-bcb8-11ea-9499-0ce5f047fa2a.png)

Reply:
Yeah, we talked about this in the other comment. Should we just change it in our copy of the spec?

Reply:
I've added a commit which modifies the spec, thanks